docs/guides/integration-prebid-server-side.md

```diff
@@ -0,0 +1,417 @@
+
+## Integration Overview: High-Level Steps
+
+You'll need to complete the following steps:
+
+1. [Complete UID2 account setup](#complete-uid2-account-setup).
+2. [Add Prebid.js to your site](#add-prebidjs-to-your-site).
+3. [Configure the UID2 module](#configure-the-uid2-module).
+
+## Complete UID2 Account Setup
+
+Complete the UID2 account setup by following the steps described in the [Account Setup](../getting-started/gs-account-setup.md) page.
+
+When account setup is complete, you'll receive your unique API key and client secret. These values are unique to you and it's important to keep them secure. For details, see [API Key and Client Secret](../getting-started/gs-credentials.md#api-key-and-client-secret).
+
+## Add Prebid.js to Your Site
+<!-- GWH "Add Prebid.js to Your Site" section is identical for client side and server side. -->
+To add Prebid.js to your site, follow the instructions in [Getting Started for Developers](https://docs.prebid.org/dev-docs/getting-started.html) in the Prebid.js documentation. 
+
+When you download the Prebid.js package, add the UID2 module by checking the box next to the module named **Unified ID 2.0**, listed under the section **User ID Modules**.
+
+When you've added Prebid.js to your site and confirmed that it's working properly, you're ready to configure the UID2 module.
+
+:::tip
+To make sure that the UID2 module is installed, find the string `uid2IdSystem` in the [`pbjs.installedModules` array](https://docs.prebid.org/dev-docs/publisher-api-reference/installedModules.html).
+:::
+
+## Configure the UID2 Module
+
+You'll need to configure the UID2 Prebid module to complete the following two actions:
+
+| Step | Action | Link to Instructions |
+| --- | --- | --- |
+| 1 | Send a server-side API call to generate a UID2 token.  | [Generating a UID2 Token on the Server](#generating-a-uid2-token-on-the-server) |
+| 2 | Store the response value, so that the Prebid module can manage token refresh as well as opt-out if needed. | [Refreshing a UID2 Token](#refreshing-a-uid2-token) |
+
+### Generating a UID2 Token on the Server
+
+To generate a token, call the [POST /token/generate](../endpoints/post-token-generate.md) endpoint.
+
+For an example, see [Sample Token](#sample-token).
+
+### Refreshing a UID2 Token
+
+There are two ways to refresh a UID2 token, as shown in the following table.
+
+| Mode | Description | Link to Section | 
+| --- | --- | --- |
+| Client refresh mode | Prebid.js automatically refreshes the tokens internally.<br/>This is the simplest approach. | [Client Refresh Mode](#client-refresh-mode) |
+| Server-only mode | Prebid.js doesn't automatically refresh the token. It is up to the publisher to manage token refresh.<br/>Choose this option if you want to use the [UID2 SDK for JavaScript](../sdks/client-side-identity.md) to refresh the token, and Prebid.js to send the token to the bid stream. | [Server-Only Mode](#server-only-mode) |
```

Comment:
Maybe mention do this to send same token to bid stream via multiple ways (e.g. google and prebid.js)
_if you want to use the UID2 SDK for JavaScript to refresh the token, and send the token to the bid stream via Prebid.js and other avenues _

Reply:
@sunnywu done -- reworked this as a bulleted list.